docs/guides/settings_guide.rst

```diff
@@ -0,0 +1,253 @@
+* ``encode`` whether or not to URL-encode the entire URL when replacing ``{url}``. Defaults to ``true``.
+
+Some items can be configured to use the proxy by default (such as basemaps and some providers). However, for most requests, they will first be tried as a normal request and only try the proxy after that request fails.
+
+providers
+---------
+``admin.providers``
+
+The providers section is the meat of any OpenSphere configuration. This provides all of the data available to the user by default. While they can certainly add their own data servers, a well-curated list is much more likely to keep users coming back.
+
+Usage:
+
+.. code-block:: javascript
+
+  {
+    "admin": {
+      "providers": {
+        "unique-id-1": {
+          "type": "geoserver", // or any provider type
+          // ... rest of provider-specific config
+        },
+        // ... more providers
+      }
+    }
+  }
+
+The design there should be simple and clear. However, let's do a specific example. The ``config/settings.json`` file in OpenSphere itself does a good job of showing how to set up the ``basemap`` provider. So we will add a couple of others:
+
+.. code-block:: json
+
+  {
+    "admin": {
+      "providers": {
+        "arc-sample-server": {
+          "type": "arc",
+          "label": "ArcGIS Online",
+          "url": "//services.arcgisonline.com/ArcGIS/rest/services/"
+        },
+        "demo-geoserver": {
+          "type": "geoserver",
+          "label": "Demo Geoserver",
+          "url": "https://demo.geo-solutions.it/geoserver/ows"
+        }
+      }
+    }
+  }
+
+plugins
+-------
+``admin.plugins``
+
+This is an object map of plugin IDs to booleans that allows you to disable a plugin entirely in config rather than having to build a new version of the application without that plugin.
+
+Say we wanted to disable KML for some reason:
+
+.. code-block:: json
+
+  {
+    "admin": {
+      "plugins": {
+        "kml": false
+      }
+    }
+  }
+
+baseProjection
+--------------
+``user.baseProjection``
+
+The ``baseProjection`` sets the default projection of the application. This projection should have a corresponding set of default map layers configured in the ``basemap`` provider. OpenSphere ships with support for EPSG:4326 and EPSG:3857 out of the box. Other projections can be added via config.
+
+Users can change this value in Settings > Map > Projection or by adding a tile layer that is in a projection other than the current projection (assuming that ``enableReprojection`` is ``false``).
+
+.. code-block:: json
+
+  {
+    "user": {
+      "baseProjection": "EPSG:4326"
+    }
+  }
+
+metrics
+-------
+``admin.metrics``
+
+OpenSphere has a metrics API that can be used to gather stats about usage. We want to stress that these metrics *not sent anywhere*. If you would like to have your metrics optionally sent somewhere, you will have to write a plugin to upload them and add that to your OpenSphere build.
```

Comment:
these metric **are** *not sent anywhere*?

Also the "optionally" part could be left out.